### DISCUSSION TITLE
Can I make custom select command for assets manager ?

### DISCUSSION THREAD

**Discussion by LamaYouzghatli** [GrapesJS/grapesjs#3891] [Q&A] [answered]
Hi I don't want to change the asset manager, I just want to change the select function. Can I do that?

**Comment by artf** [comment, marked as answer]
You can have a custom selection strategy via [`AssetManager.open`](https://grapesjs.com/docs/api/assets.html#open), but keep in mind that if your goal is changing the select strategy for something like an Image component, you have to extend its view (`onActive` method) as it has already its [own strategy](https://github.com/artf/grapesjs/blob/29ff3a4b7c29e9466c75235fd9029e9d76ef3259/src/dom_components/view/ComponentImageView.js#L63-L79).
